Assets/Scripts/Dialogue/InkManager.cs

```diff
@@ -113,13 +108,10 @@ private void InstantiateChoiceButtons()
             button.transform.SetParent(choiceButtonParent, false);
         }
     }
-    /// <summary>
-    /// Tags are currently unused, I am planning to use them for changing the portraits depending on emotion but this will only be added when CharacterManager is implemented
-    /// </summary>
-    /// <param name="pText"></param>
 
     private void HandleTextTags(string pText)
     {
+        string speakerID = "B";
```

Comment:
Could you make it more clear what "B" is. Maybe rename it to "Character_B" or something